lib/vsc/filesystem/posix.py

```diff
@@ -257,16 +257,14 @@ def _local_filesystems(self):
             # returns [('rootfs', '/', 2051L, 'rootfs'), ('ext4', '/', 2051L, '/dev/root'),
             # ('tmpfs', '/dev', 17L, '/dev'), ...
             self.localfilesystemnaming = ['type', 'mountpoint', 'id', 'device']
+            # do we need further parsing, eg of autofs types or remove pseudo filesystems ?
+            if self.ignorefilesystems:
+                currentmounts = [x for x in currentmounts if not x[2] in OS_LINUX_IGNORE_FILESYSTEMS]
```

Comment:
This seems odd. The comment above states that each `x` will be a tuple (type, mount point, id, device), so `x[2]` would be the ID, no? Maybe I am overlooking something :)

Reply:
yes, because the next line orders them in that way

Reply:
as you can see I now loop over currentmounts instead of self.localfilesystems

Reply:
only self.localfilesystems is in that particular order

Reply:
Ah yes.